cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -24,13 +24,107 @@
     {'Name': files_list[0]['Name'],
      'Collect Contents?': not files_list[0]['Collect Contents?']}]
 
+TENANT_NAME = "tenant_{}".format(fauxfactory.gen_alphanumeric())
+
+PRODUCT_FEATURES = [
```

Comment:
This makes for pretty unreadable parametrization of the tests. The first two items have 6 strings in common, and then the rest have a different set of 6 strings in common. Only the last two strings change in either case.

Need to at a minimum remove the duplication here, but I think we need to brainstorm on a better way to parametrize this information in the first place.